### ISSUE TITLE
[Rule Rsch] Tool/Method for testing Touch Target Size (WCAG 2.2 SC 2.5.8)

### ISSUE THREAD

**Issue by marcjohlic** [IBMa/equal-access#694]
### Describe what needs to be accomplished

Designers, Developers, Testers and/or Product teams in general needing to meet WCAG 2.2 ([2.5.8 Target Size (Minimum)](https://www.w3.org/TR/WCAG22/#target-size-minimum)) will need an easy way to test the Target size of actionable objects/components in their web sites & apps to meet [2.5.8 Target Size (Minimum)](https://www.w3.org/TR/WCAG22/#target-size-minimum).  

Possible tests/solutions could be:

- Checker will return violations for touch targets that do not meet the minimum requirement (24 x 24 CSS pixels)
- Checker will visually highlight touch targets and display their size
- Checker will test against WCAG (minimum), (Android, Apple, and HMI as a bonus)  

#### Rule Research

- [ ] First step is to perform the new rule research on [2.5.8 Target Size (Minimum)](https://www.w3.org/TR/WCAG22/#target-size-minimum)

### Why does this need to be accomplished
Teams will need an easy way to test the Target size of actionable objects/components in their web sites & apps to meet [2.5.8 Target Size (Minimum)](https://www.w3.org/TR/WCAG22/#target-size-minimum).  

### Related issues or references
This was an internal [Parking Lot submission](https://ibm.ent.box.com/notes/880390813501)

### Supporting notes and details

- Expect this to be a success criteria
- Can we just flag violations -not necessarily add a visualization
- Could start with WCAG 2.2 preview
- Create a new issue in external repo - new rule research
  - what is the target?

**Comment by aliunwala** [comment]
Triage:
This is from WCAG 2.2  so this can be picked up at a later date

**Comment by marcjohlic** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Equal-Access-61ae71f9f8b4ec001e136462/issues/IBMa/equal-access/694?planning-poker) with ZenHub @aliunwala @drjoho @mbgower @philljenkins @shunguoy @tombrunet

**Comment by MHoov** [comment]
Closing - we have an existing rule from WCAG 2.2 work in 2024.
